packages/xrpl/src/models/ledger/XChainOwnedCreateAccountClaimID.ts

```diff
@@ -0,0 +1,60 @@
+import { XChainBridge } from '../common'
+
+import BaseLedgerEntry from './BaseLedgerEntry'
+
+/**
+ * The XChainOwnedCreateAccountClaimID ledger object is used to collect attestations
```

Comment:
Is it possible to @link to what attestations are? (seems like a jargon word when people are figuring things out - maybe it's worth making a list of these words for when xrpl.org has docs for them, then going in and adding links?)

Reply:
There are two types of attestations so there isn't really one link for "attestations"

Reply:
Can we add a parent page for both kinds? (Since that seems even more worth knowing about since there's complexity hidden there)

Reply:
I'll ask the docs team to add it to this terminology section: https://opensource.ripple.com/docs/xls-38d-cross-chain-bridge/cross-chain-bridges/#terminology
A whole page seems overkill.

Reply:
cc @oeggert